WooCommerce/Classes/ViewRelated/Orders/Order Details/Address Edit/EditAddressForm.swift

```diff
@@ -252,9 +252,37 @@ private extension EditAddressForm {
 
 #if DEBUG
 
+import struct Yosemite.Order
 import struct Yosemite.Address
 
 struct EditAddressForm_Previews: PreviewProvider {
+    static let sampleOrder = Order(siteID: 123,
+                                   orderID: 456,
+                                   parentID: 2,
+                                   customerID: 11,
+                                   number: "789",
+                                   status: .processing,
+                                   currency: "USD",
+                                   customerNote: "",
+                                   dateCreated: Date(),
+                                   dateModified: Date(),
+                                   datePaid: Date(),
+                                   discountTotal: "0.00",
+                                   discountTax: "0.00",
+                                   shippingTotal: "0.00",
+                                   shippingTax: "0.00",
+                                   total: "31.20",
+                                   totalTax: "1.20",
+                                   paymentMethodID: "stripe",
+                                   paymentMethodTitle: "Credit Card (Stripe)",
+                                   items: [],
+                                   billingAddress: sampleAddress,
+                                   shippingAddress: sampleAddress,
+                                   shippingLines: [],
+                                   coupons: [],
+                                   refunds: [],
+                                   fees: [])
```

Comment:
nit: Remember we have `.fake().copy()` 😅

Reply:
Right! But it requires linking Fakes.framework to main app target which we haven't done yet.
What do you think? Keep it in tests for now or start using in SwiftUI previews?

Reply:
I totally forgot that these are not tests 🤦